docs/source/install/prerequisites.rst

```diff
@@ -205,30 +209,73 @@ Make sure that RabbitMQ is running with:
     Remove the mentioned option from the file ``/usr/libexec/rabbitmq/rabbitmq-env`` and restart the server.
     If you still have trouble getting RabbitMQ to run, please refer to the detailed instructions provided on the `website of RabbitMQ itself for generic Unix systems <https://www.rabbitmq.com/install-generic-unix.html>`_.
 
+
 .. _details_wsl:
 
 Windows Subsystem for Linux (Ubuntu)
 ====================================
 
-The guide for Ubuntu above can be followed but there are a few things to note:
-
-#. The `windows native RabbitMQ <https://www.rabbitmq.com/install-windows.html>`_
-   should be installed and started,
-   since the version in WSL Ubuntu 18.04 does not work properly.
```

Comment:
Does RabbitMQ work in WSL2? When I set up my current system (~August), RMQ on WSL1 definitely didn't work properly, so I'm not sure if this note should be kept.

Reply:
This is 

> 
> 
> Does RabbitMQ work in WSL2? When I set up my current system (~August), RMQ on WSL1 definitely didn't work properly, so I'm not sure if this note should be kept.

Probably works only with WSL2 as most things should work, after all, it is effectively a virtual machine. With WSL1 I have to use RMQ from the windows side.

Reply:
Okay. I can keep it in. I removed it because I have installed it on WSL1 (version 1909 build 18363.476) and 2 (latest version on Insider slow ring). And it all worked, except for some transport tests.